.github/workflows/docker-publish.yml

```diff
@@ -14,6 +14,7 @@ permissions:
 env:
   AWS_EC2_INSTANCE_ID: i-0960f9b8de8285aed
   AWS_REGION: us-east-1
+concurrency: ${{ github.ref }}
```

Comment:
I don't think this is correct. This would still run multiple docker builds upon different references. We want to ensure exactly 1 docker build is happening at a given time so the build machine doesn't run out of resources.

Reply:
Ok, let's make it something static then.